db/migrate/20170421154535_create_default_price_groups_for_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateDefaultPriceGroupsForUsers < ActiveRecord::Migration
+  def up
+    User.find_each do |user|
+      # does nothing if user_based_price_groups is not turned on
+      user.create_default_price_group!
+    end
+  end
+  def down
+    raise ActiveRecord::IrreversibleMigration
```

Comment:
I think it would be safe to delete any user price group memberships that are part of the base or external. Might need to think a little more about that, though.